content/overview/pricing/rates.md

```diff
@@ -15,44 +15,42 @@ cloud.gov fees are broken into three components:
 
-| Package | What's included? | Base annual fee\* | Application resource usage | Managed services available |
+| Package | What’s included? | Base annual fee\* | Resource usage quota | Managed services available |
 | --- | --- | --- | --- | --- |
-| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov, no paperwork required. **Application resource usage is capped at 1GB.** | Free | Free | Only free services |
+| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov for a limited time with no paperwork required. **Application resource usage is capped at 1GB.** | Free | Free | Only free services |
 | **Prototyping** | Suitable for many teams to deploy apps, though limited to the `*.app.cloud.gov` domain. Access control can be delegated to teams. No production data allowed. Usually purchased per agency/department. | $15K |  ~$99/GB/month | All\** |
 | **Open Data** | One public-facing Open Data (no confidentiality risk assessed) system, including all the spaces needed and DNS support. **This account is limited to 2GB of memory usage per month.** | $10K | ~$99/GB/month | All\** (up to $2500/year) |
 | **FISMA Low** | One public-facing FISMA Low system, including all the spaces needed and DNS support. | $20K | ~$99/GB/month | All\** |
 | **FISMA Moderate** | One public-facing FISMA Moderate system, including all the spaces needed and DNS support. Additional support for FISMA Moderate data requirements. | $90K | ~$99/GB/month | All\** |
 
 \*Access agreements are severable, and the access package fees are invoiced monthly.
 
-\*\*We aren't yet charging for paid services. See [managed services](#managed-services) for details.
+\*\*We aren’t yet charging for paid services. See [managed services](#managed-services) for details.
 
 ## Resource usage quota
 
-You will be charged for the **quota** you set for your resource usage. This is only based on the memory, in MB of RAM, your reserve for your applications.
+Along with the access package you select, you will be charged for the **quota** you set for your resource usage. The quota is the memory, in MB of RAM, your reserve for your applications. Unused quotas are not refunded, and they do not ‘roll over’ month to month.
 
-- Sandbox: Resource usage is capped to 1 GB of RAM per month.
-- All other packages: $0.0033 per MB reserved per day (usually this results in $99/GB/month)
-
-Quotas are monitored daily and are invoiced monthly. Unused quotas are not refunded, and they do not 'roll over' month to month.
+Quotas are monitored daily and are invoiced monthly at these rates:
 
-We require your resource usage quota to **be set in advance**. This helps you control the maximum amount that your team will spend, and helps us avoid over-provisioning resources that won't be used.
+- Sandbox: Free, with resource usage capped at 1 GB of RAM per month.
```

Comment:
@brittag Is the package free and capped, or is it $99/GB/mo and capped? (In other words, capped at $198, but we wouldn't say it quite like that.)